components/combobox.stories.tsx

```diff
@@ -318,3 +319,261 @@ export const MultipleSelection: Story = {
     );
   },
 };
+
+export const WithDisplay: Story = {
```

Comment:
can we change this to `WithCustomRenderer` ? instead of showing it "With Display"